Bottom: e158cfb023f019d4ef91f72e39f049c35cf5993e
Top:    cf88942daee97c712f00f5b3c85efe180c90b3d3
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-05-11 10:41:02 -0700

rstream: Clarify use of async test option

Signed-off-by: Sean Hefty <sean.hefty@intel.com>


---

diff --git a/examples/rstream.c b/examples/rstream.c
index a52a262..b089c2a 100644
--- a/examples/rstream.c
+++ b/examples/rstream.c
@@ -561,7 +561,7 @@ int main(int argc, char **argv)
 			printf("\t[-p port_number]\n");
 			printf("\t[-T test_option]\n");
 			printf("\t    s|sockets - use standard tcp/ip sockets\n");
-			printf("\t    a|async - asynchronous operation\n");
+			printf("\t    a|async - asynchronous operation (use poll)\n");
 			printf("\t    b|blocking - use blocking calls\n");
 			printf("\t    n|nonblocking - use nonblocking calls\n");
 			printf("\t    v|verify - verify data\n");
